packages/backend/src/message.ts

```diff
@@ -53,6 +53,7 @@ export interface SystemConnected {
 
 export interface SystemInterval extends BestBlock {
   msg: 'system.interval';
+  network_state: Types.NetworkState;
```

Comment:
This should be a `Maybe<Types.NetworkState>`. Older versions of Substrate won't be sending those, Backend shouldn't flip on `undefined`.